### PR TITLE
fixed test cases and a bit of refactoring

### PR DESCRIPTION
**Pull request by matthewhung09**

### CODE REVIEW

File: backend/backend.js [hunk 1]
```diff
@@ -338,7 +350,7 @@ app.post("/reset/:userId/:token", async (req, res) => {
 app.post("/auth/remove", checkUser, async (req, res) => {
   console.log("k0w46un94");
   const user_id = req.body._id;
-  const user = await userServices.deleteSpotifyAccess(user_id);
+  const user = await userServices.updateRefresh(user_id, "");
```
Comment (gscotti6950) [inline backend/backend.js:353]: Oh nice. I forgot we already made a function for this. Nice catch!

File: backend/models/post-services.js [hunk 2]
```diff
@@ -18,17 +18,13 @@ function getDbConnection() {
   return dbConnection;
 }
 
-async function getPostsByLocation(lat, long, only_coords) {
```
Comment (gscotti6950) [inline backend/models/post-services.js:21]: I'm meant to get rid of this, didn't end up needing it. Nice job!

**Review by gscotti6950 — COMMENTED** [review]
Awesome job! Lot of good and necessary clean up.

**Comment by naomidonato** [comment]
Good job. These features are functional.

**Pull request merged**